tests/performance/utils.js

```diff
@@ -21,13 +21,67 @@ function median( array ) {
  * @return {string} Result file name.
  */
 function getResultsFilename( fileName ) {
-	const prefixArg = process.argv.find( ( arg ) => arg.startsWith( '--prefix' ) );
-	const fileNamePrefix = prefixArg ? `${prefixArg.split( '=' )[1]}-` : '';
+	const prefixArg = process.argv.find( ( arg ) =>
+		arg.startsWith( '--prefix' )
+	);
+	const fileNamePrefix = prefixArg ? `${ prefixArg.split( '=' )[ 1 ] }-` : '';
 	const resultsFilename = fileNamePrefix + fileName + '.results.json';
 	return resultsFilename;
 }
 
+/**
+ * Returns time to first byte (TTFB) using the Navigation Timing API.
+ *
+ * @see https://web.dev/ttfb/#measure-ttfb-in-javascript
+ *
+ * @return {Promise<number>}
+ */
+async function getTimeToFirstByte() {
+	return page.evaluate(
+		() =>
+			new Promise( ( resolve ) => {
+				new PerformanceObserver( ( entryList ) => {
```

Comment:
> Alternatively, instead of creating a new dedicated listener, in Gutenberg we use directly the performance data already available

Yep, that's what I actually wanted to do here as well. Not sure why I copied this more complicated method.

See 31b203f1b3ea1e4d9987e62ad67c0cddc7135720